pocs/focuser/focuser.py

```diff
@@ -228,6 +244,29 @@ def autofocus(self,
             else:
                 merit_function_kwargs = {}
 
+        if take_dark:
+            image_dir = self.config['directories']['images']
+            start_time = current_time(flatten=True)
+            file_path = "{}/{}/{}/{}/{}.{}".format(image_dir,
+                                                   'focus',
+                                                   self._camera.uid,
+                                                   start_time,
+                                                   "dark",
+                                                   self._camera.file_extension)
+            self.logger.debug('Taking dark frame {} on camera {}'.format(file_path, self._camera))
+            try:
+                dark_thumb = self._camera.get_thumbnail(seconds,
+                                                        file_path,
+                                                        thumbnail_size,
+                                                        keep_files=True,
```

Comment:
Should this always be `True` or should it be the `keep_files` param? Same with `dark` below, should maybe be `autofocus_take_dark`

Reply:
I'm causing confusion here by using the same argument name for two different purposes. In `get_thumbnail` `keep_files` simply determines whether the one FITS file it creates gets immediately deleted or not, but in the `autofocus` and `_autofocus` methods `keep_files` means keep _all_ files when `True`, and keep _some_ files when `False`.  The dark frame is one of the files I decided it should always keep, along with the initial and final focus images.

Reply:
Argument of `get_thumbnail` changed from `keep_files` to `keep_file` to slightly reduce confusion, docstring rewritten.